### DISCUSSION TITLE
Switching from double to single screening - merging .asreview project files

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1912] [Q&A]
Dear all, 

We have applied independent screening in ASReview. We have two datasets that have the same prior knowledge, classifier, and feature extractor. Based on good inter-rater agreement, we want to continue with single screening. 
Ideally, we would like to merge the datasets to not lose any information because some abstracts have only been seen by one of the reviewers.
**We see 2 options right now.** 
**1)** Export the two independent projects as an Excel file, and merge them in Excel. Afterward, creating a new project in ASReview with this merged Excel file. 
**2)** Continuing with the project file that is more advanced (i.e., has more labelled records). We would still full-text screen the additional abstracts of the other project file, but continue abstract screening with one file without merging. 

Ideally, we prefer Option 1. However, we are unsure whether this is a good choice because we will lose information on the model configurations. **Thus, my questions are:** 

- To what extent would Option 1 affect the results?
- Do you think this would be a transparent and reproducible research practice, given we save the intital two .asreview project files that we merge in Excel?
- Do you see a better method for merging the files? 
- Do you see any other bottlenecks or problems with Option 1? 

Many thanks in advance for any input and help! Much appreciated!

**Comment by Rensvandeschoot** [comment]
Every iteration is trained from scratch (see https://doi.org/10.3390/app14093842), so when you combine both files and create a new project, the next results will be based on a combination of labels from both screeners. With [Datatools](https://doi.org/10.3390/app14093842) you could merge both files and deal with conflicting labels.

**Reply by Morle2104** [reply]
Yes, great, many thanks. We merged the xlsx ASReview output files in Excel, and dealt with conflicting labels. Now I uploaded an Excel file with the column label_included and created a new project. ASReview recognizes all labels correctly as prior knowledge, however there are two strange things that happened. 
Number 1, in Analytics it gives the number 1.599 for 'irrelevant records since last relevant', but I have no idea where this number comes from. I uploaded the Excel and made a new project and have not started screening again. In the Excel with which I created the new project, there are only columns related to the articles (pubmed id, abstract, title etc.) and the label_included column. However, it is still the structure of the Excel output file of ASReview in the sense that all as relevant labelled articles are at the top and all as irrelevant at the bottom. 
The second strange thing is that I uploaded the same Excel sheet a couple of times, and everytime it gives me a different first abstract to screen. However, given that I chose query strategy 'maximum', it should always be the same abstract. 
I checked the Excel sheet and tried to modify some things that I thought caused the problem (e.g., I took the record_id column out), but that did not solve it.

**Reply by Rensvandeschoot** [reply]
continued in https://github.com/asreview/asreview/discussions/1924
